src/features/pages/markets/etf/other-markets/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import MarketsMainSlider from '../carousel-markets'
+import { ETFOtherMarkets } from './data'
+import Container from 'features/components/atoms/container'
+import Flex from 'features/components/atoms/flex-box'
+import Typography from 'features/components/atoms/typography'
+import { Localize } from 'components/localization'
+
+const OtherMarkets = () => {
+    return (
+        <Container.Fluid as="section" padding_block="20x">
+            <Flex.Box direction="col" gap="12x">
+                <Typography.Heading
+                    as="h1"
```

Comment:
It seems to me that not heading of hero, so we can use h3 or h4 instead for better SEO